src/components/Form/UploadDocumentForm.jsx

```diff
@@ -24,53 +23,73 @@ const UploadDocumentForm = () => {
   const { session } = useSession();
   const { state, dispatch } = useStatusNotification();
   const [docType, setDocType] = useState('');
+  const [verifyFile, setVerifyFile] = useState(false);
+  const [expiryDate, setExpiryDate] = useState('');
+  const [description, setDescription] = useState('');
+  const [file, setFile] = useState('');
```

Comment:
I think `null` would be a better initial state than an empty string since file will set to a File Object when a file is chosen.

Reply:
fixed